templates/shared/issuelist.tmpl

```diff
@@ -67,7 +67,7 @@
 						</a>
 					{{end}}
 					{{if .Ref}}
-						<a class="ref" {{if $.RepoLink}}href="{{$.RepoLink}}{{index $.IssueRefURLs .ID}}"{{else}}href="{{AppSubUrl}}/{{.Repo.OwnerName}}/{{.Repo.Name}}{{index $.IssueRefURLs .ID}}"{{end}}>
+						<a class="ref" href="{{index $.IssueRefURLs .ID}}">
```

Comment:
```suggestion
 						<a class="ref" {{if $.RepoLink}}href="{{index $.IssueRefURLs .ID}}"{{else}}href="{{AppSubUrl}}/{{.Repo.OwnerName}}/{{.Repo.Name}}{{index $.IssueRefURLs .ID}}"{{end}}> 
```

Reply:
Just to confirm this a last time: `.RepoLink` holds the information whether `{index $.IssueRefURLs .ID}}` holds the full URL or not? That makes sense then.

Reply:
Right, that's my understanding from a couple of tests (it actually holds the RepoLink, but this is only set on the `<owner>/<repo>/issues` page)